signal_scanner_bot/signal.py

```diff
@@ -12,8 +12,18 @@
 ################################################################################
 # Send message
 ################################################################################
-def send_message(message: str, recipient: Optional[str], group: bool = False):
-    recipient_args = ["-g", str(recipient)] if group else [str(recipient)]
+def send_message(message: str, recipient: Optional[str]):
+    recipient = recipient or ""
+    if recipient.endswith("=="):
```

Comment:
It looks we could also add `and len(recipient) == 24` to be more sure that the configured value is a group ID. I made a few groups to double check if they always end in a `==` and it looks like they do, and they're also always 24 characters long.

Reply:
Great point! I'll add that in